src/ubuntu/stage-upgrade.md

```diff
@@ -0,0 +1,11 @@
+### Stage the upgrades
+"Staging an upgrade" is a process in which you tell your node to download the upgrade files and prepare them, so that they can automatically be applied at the pre-defined activation point.
+
+#### Upgrade to casper-node v1.1.0
+For this upgrade, to `casper-node v1.1.0`, the activation point is `Era 166`. In order to not have points deducted for your Testnet reward score, you have to make sure you have properly staged the upgrade well ahead of the activation point, so that your node will be upgraded on time.
```

Comment:
@mrkara apparently it's best to stage the upgrade prior to even starting the node.  Let's see if we can integrate the instruction to stage the upgrade into the installation process between setting up the initial software version and actually starting the node.